edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/Program.cs

```diff
@@ -102,8 +103,10 @@ static async Task<int> MainAsync(IConfigurationRoot configuration)
                 await protocolHead.StartAsync();
                 await Task.WhenAny(cts.Token.WhenCanceled(), renewal.Token.WhenCanceled());
                 logger.LogInformation("Stopping the protocol heads...");
-                await Task.WhenAny(protocolHead.CloseAsync(CancellationToken.None), Task.Delay(TimeSpan.FromSeconds(10), CancellationToken.None));
+                await protocolHead.CloseAsync(CancellationToken.None);
```

Comment:
Yeah, I removed it because we were already waiting for 60 seconds before shutting the app down in ShutdownHandler so another artificial wait here doesn't make much sense? 

I also wanted the Db store provider to be closed only after the protocol heads are closed so that backup and restore occurs only after the protocol heads are closed to prevent cases where we don't backup some messages.